c/zss.c

```diff
@@ -389,6 +394,25 @@ static JsonObject *readServerSettings(ShortLivedHeap *slh, const char *filename)
   return mvdSettingsJsonObject;
 }
 
+static void initZssBackgroundTasks(HttpServer *server) {
+  zowelog(NULL, LOG_COMP_ID_MVD_SERVER, ZOWE_LOG_DEBUG,"initZssBackgroundTasks\n");  
+  // register background handler
+  STCBase *base = server->base;
+  stcRegisterModule(
+    base,
+    STC_MODULE_ZSS,
+    server,
+    NULL,
+    NULL,
+    NULL,
+    processZssBackgroundHandler
+  );
+
+  for(int i=0; i < N_TASK_TABLE_ENTRIES; i++) {
+    task_list[i].id = 0;  /* initialise */
```

Comment:
Where does this live? Is this global somewhere?

Reply:
yes this is global right now